### DISCUSSION TITLE
關於大廳Lobby

### DISCUSSION THREAD

**Discussion by Liteya** [GpointChen/FFXIVChnTextPatch-GP#344] [General]
額外做了一份對應標準字體的Lobby 檔案 不過不確定有沒有哪邊寫錯
就參考一下?
[Lobby.zip](https://github.com/GpointChen/FFXIVChnTextPatch-GP/files/11563851/Lobby.zip)

12行的遊戲版本的部分就看有沒有需要做標記 沒有的話就直接把中文化標記拿掉就好

**Comment by GpointChen** [comment]
* 有部分牽涉到指令碼的部分沒有修改到長度，例如1200-1221行，我手動修過了
* 將大區統一修改為資料中心
* 因為牽涉到大量遊戲內用語和指令碼，而且大廳說明文字中有說明伺服器=世界，故伺服器不特別修改
* 只有零星「＝」而不會完全無法顯示中文的地方選擇保留中文

修改內容可參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/commit/95f6320f99f3782fe27a7e147cdb8a3e78cb9abf

預計待會會發一個新的release。

**Reply by Liteya** [reply]
了解
